MapboxNavigation/NavigationView.swift

```diff
@@ -69,7 +69,7 @@ open class NavigationView: UIView {
     }
     
     private enum Actions {
-        static let cancelButton: Selector = #selector(NavigationView.cancelButtonTapped(_:))
+//        static let cancelButton: Selector = #selector(NavigationView.cancelButtonTapped(_:))
```

Comment:
Seems like this class has some placeholder/WIP stuff going on (methods below as well).